net-im/skype-dbus-mock/skype-dbus-mock-0_p20181011.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2018 Gentoo Authors
```

Comment:
your naming scheme is wrong, `_p` is for cherry picked patch changes

Reply:
What is the naming scheme supposed to be for a package that has no version and is based on a git commit from a specific date?

Reply:
Would _pre be more appropriate?